kani-compiler/src/kani_compiler.rs

```diff
@@ -101,9 +102,18 @@ impl KaniCompiler {
         let all_stubs = stubbing::collect_stub_mappings(tcx);
         if all_stubs.is_empty() {
             FxHashMap::default()
-        } else if let Some(harness) = self.args.as_ref().unwrap().get_one::<String>(parser::HARNESS)
+        } else if let Some(harnesses) =
+            self.args.as_ref().unwrap().get_many::<String>(parser::HARNESS)
         {
-            find_harness_stub_mapping(harness, all_stubs).unwrap_or_default()
+            let mappings = filter_stub_mapping(harnesses.collect(), all_stubs);
+            if mappings.len() > 1 {
+                tcx.sess.err(format!(
+                    "Failed to apply stubs. Harnesses with stubs must be verified separately. Found: `{}`",
```

Comment:
```suggestion
                    "Failed to apply stubs. Harnesses with stubs must be verified separately. The `--harness` option matched multiple harnesses: `{}`",
```
or anything shorter you can think of.

Reply:
Yeah, I was trying to keep it short. Also, I am planning to get rid of this restriction sooner rather than later. Would you mind if I just leave it as is for this PR?

Reply:
Sure.